src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/VetCreator.java

```diff
@@ -91,9 +88,11 @@ public List<String> createRow(final long location, final VariantContext variant,
         for ( final VetFieldEnum fieldEnum : VetFieldEnum.values() ) {
             if (fieldEnum.equals(VetFieldEnum.location)) {
                 row.add(String.valueOf(location));
-            } else if (fieldEnum.equals(VetFieldEnum.sample_id)) {
+            }
```

Comment:
The `if/else` reformatting here and below is actually contrary to what my IntelliJ would do; is there a reason for this?

Reply:
Yeah, that's me. I followed a different style guide in a former life and don't like the way it looks like that. But I'll stick with what IJ does.